source-facebook-marketing-native/source_facebook_marketing_native/client.py

```diff
@@ -250,6 +304,7 @@ async def _fetch_resource_data(
                 self.log,
                 url,
                 params=request_params,
+                should_retry=self._make_should_retry(params),
```

Comment:
Params is not the object being passed into self.http.request, so is it going to work when you modify the captured type in the callback?